checker.h

```diff
@@ -0,0 +1,52 @@
+/* Include Headerfiles													*/
+/* ===================													*/
+#include <stdio.h>
+#include <assert.h>	
+#include <stdbool.h>
+/* ===================													*/
+/* Macros and define values */
+/* ===================													*/
+#define MAXCHARGERATE 0.5
+#define MAXTEMP 45.0
+#define MINTEMP 0
+#define MAXSOC 80.0
+#define MINSOC 20.0
+#define German 0
+#define English 1
+
+int language=German;
```

Comment:
Instead of this, you could have 2 pointers that point to the selected  BMSattribute... Or Displayin... arrays. Then you don't need to repeat the if statement to select the language.